bindings/rust-bindings/build.rs

```diff
@@ -2,7 +2,8 @@ use std::env;
 use std::path::PathBuf;
 
 fn main() {
-    let manifest_dir = env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR is not set");
+    let manifest_dir = env::current_dir().expect("Failed to get current directory");
+    // let manifest_dir = env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR is not set");
```

Comment:
Did you mean to leave this commented out?

Reply:
I'll remove this for now - these changes are not needed for the fix.